php/WP_CLI/Runner.php

```diff
@@ -376,31 +376,22 @@ public function find_command_to_run( $args ) {
 			$subcommand = $command->find_subcommand( $args );
 
 			if ( ! $subcommand ) {
-				if ( count( $cmd_path ) > 1 ) {
-					$child       = array_pop( $cmd_path );
-					$parent_name = implode( ' ', $cmd_path );
-					$suggestion  = $this->get_subcommand_suggestion( $child, $command );
 
-					if ( 'network' === $parent_name && 'option' === $child ) {
-						$suggestion = 'meta';
-					}
+				if ( $this->wp_exists() && $this->wp_is_readable() ) {
+					$this->load_wordpress();
 
+					// Suggest appropriate commands for taxonomy or post type.
+					if ( $this->is_taxonomy( $cmd_path[0] ) ) {
+						WP_CLI::error( "Did you mean 'wp term <command>' ?" );
```

Comment:
Just like the existing "Did you mean" message below, this should just return a string, not use `WP_CLI::error()`.